src/main/kotlin/com/getkeepsafe/dexcount/PrintOptions.kt

```diff
@@ -16,13 +16,13 @@
 
 package com.getkeepsafe.dexcount
 
-final class PrintOptions {
-    def includeClasses = false
-    def includeMethodCount = true
-    def includeFieldCount = false
-    def includeTotalMethodCount = false
-    def teamCityIntegration = false
-    def printHeader = false
-    def orderByMethodCount = false
-    def int maxTreeDepth = Integer.MAX_VALUE
+class PrintOptions {
+    var includeClasses: Boolean? = false
```

Comment:
This is internal to the plugin, so I don't think these need to be nullable at all.  Let's remove the `?` from field types in this class.